common/objectives/00_plots.txt

```diff
@@ -770,8 +770,7 @@ plot_kill_character = {
 	target_potential = {
 		# Warcraft
 		is_untouchable_trigger = no
-		is_humanoid_trigger = yes
-		has_invader_title_trigger = no
+		sits_on_frozen_throne_trigger = no
```

Comment:
These changes are not present on current master. Have you been merging other branches into your own or basing it on a non-master branch?